pkgs/development/tools/misc/docopts/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
```

Comment:
Did you try using buildGoModule? It does not require the deps.nix and is preferred.

Reply:
I did, the source in an old-style go project, that doesn't have a mod file, so we're stuck with GoPackage.